source/common/upstream/outlier_detection_impl.h

```diff
@@ -248,6 +248,40 @@ struct DetectionStats {
   ALL_OUTLIER_DETECTION_STATS(GENERATE_COUNTER_STRUCT, GENERATE_GAUGE_STRUCT)
 };
 
+// Names used in runtime configuration.
+const std::string max_ejection_percent_runtime = "outlier_detection.max_ejection_percent";
```

Comment:
I would use `constexpr absl::string_view` here and make all the names use `ProperCase` to reflect that they're string constants

Reply:
Done. Thanks.